src/components/DividedCard/index.tsx

```diff
@@ -3,58 +3,63 @@ import {
-  Props:
-  @contentFirst : Contenido a ser mostrado en la tarjeta de arriba/izquierda, dependiendo de la orientación
-  @contentSecond : Contenido a ser mostrado en la tarjeta de abajo/derecha, dependiendo de la orientación
-  @colorFirst : Color de la tarjeta de arriba/izquierda, dependiendo de la orientación
-  @colorSecond : Color de la tarjeta de abajo/derecha, dependiendo de la orientación
-  @percentageFirst : Porcentaje del total de la altura/anchura (dependiendo de la orientación) que abarcará la primera tarjeta
-  @percentageFirst : Porcentaje del total de la altura/anchura (dependiendo de la orientación) que abarcará la segunda tarjeta
-  @vertical : Booleano que indica la orientación del componente. Si es true, la orientación es vertical. Si es false, es horizontal
-  @overlap : Booleano que indica si la tarjeta de arriba/izquierda debería sobreponerse a la otra
+  DividedCard: Componente que representa una tarjeta dividida en dos secciones distintas
+*    IDividedCard:
+*     @contentFirst : Contenido a ser mostrado en la tarjeta de arriba/izquierda, dependiendo de la orientación
+*     @contentSecond : Contenido a ser mostrado en la tarjeta de abajo/derecha, dependiendo de la orientación
+*     @colorFirst : Color de la tarjeta de arriba/izquierda, dependiendo de la orientación
+*     @colorSecond : Color de la tarjeta de abajo/derecha, dependiendo de la orientación
+*     @percentageFirst : Porcentaje del total de la altura/anchura (dependiendo de la orientación) que abarcará la primera tarjeta
+*     @percentageFirst : Porcentaje del total de la altura/anchura (dependiendo de la orientación) que abarcará la segunda tarjeta
+*     @vertical : Booleano que indica la orientación del componente. Si es true, la orientación es vertical. Si es false, es horizontal
+*     @overlap : Booleano que indica si la tarjeta de arriba/izquierda debería sobreponerse a la otra
+*     @basePropsFirst?: Props base para Flex. Referencia: https://chakra-ui.com/docs/components/layout/flex
+*     @basePropsSecond?: Props base para Center. Referencia: https://chakra-ui.com/docs/components/layout/center
+*     @containerProps?: Props base para Center. Referencia: https://chakra-ui.com/docs/components/layout/center
+}
 */
 
-export const DividedCard = (props: IDividedCard) => {
-  return (
-
+export const DividedCard = (props: IDividedCard) => (
     <Flex
-      boxShadow="0px 5px 5px 0px rgba(0,0,0,0.30)"
+      boxShadow="general"
       borderRadius="general"
       direction={(props.vertical)? "column" : "row"}
       w="100%"
       h="100%"
       bg={(props.overlap)? props.colorSecond:""}
-      
+      {...props.containerProps}
     >
       <Center
         zIndex={(props.overlap)? 2:1}
         position="relative"
-        boxShadow={(props.overlap)? "0px 5px 5px 0px rgba(0,0,0,0.40)" : ""}
-        h={(props.vertical)? props.percentageFirst : "auto"}
+        boxShadow={(props.overlap)? "general" : ""}
```

Comment:
Si tu ternearia tiene como valor secundario un "" (O simplemente un vacío, o nada), puedes reemplazarlo por una ["evaluación de corto circuito"](https://stackoverflow.com/questions/40682064/what-does-operator-indicate-with-this-props-children-react-cloneelemen). 

Quedaría algo así: 
`{props.overlap && "general"}`

Reply:
Al intentar hacer este cambió me sale este error: 
![Captura de pantalla de 2022-04-14 10-32-18](https://user-images.githubusercontent.com/61522818/163423772-f15e602a-d651-4559-b60b-967a2d0fbd95.png)

Reply:
Muchas gracias por la aclaración, Uzi! Que raro lo del error, pero no pasa nada, lo resolviste de forma eficiente! WUJUU